test/src/test/java/org/corfudb/integration/CorfuStoreBrowserIT.java

```diff
@@ -381,7 +408,8 @@ public void browserRegistryTableTest() throws
         runtime = createRuntime(singleNodeEndpoint);
         CorfuStoreBrowser browser = new CorfuStoreBrowser(runtime);
         // Invoke listTables and verify table count
-        Assert.assertEquals(2, browser.printTableInfo(TableRegistry.CORFU_SYSTEM_NAMESPACE,
+        final int three = 3; // keep nutty code check from crying about magic number
```

Comment:
Remove comment, or escape the check with an ignore annotation.

Reply:
Can you please share the ignore annotation so I can remove this comment?

Reply:
Btw this magic number check does not make much sense in test code, I think the magic number checking should not be done for test files. What do you say?

Reply:
You can use `@SuppressWarnings("checkstyle:magicnumber")` .